codegen.yml

```diff
@@ -16,6 +16,7 @@ generates:
         StarRatings: api-server/src/models/types#StarRatings
         Company: api-server/src/models/types#Company
         JobAd: api-server/src/models/types#JobAd
+        JobApplication: api-server/src/models/types#JobApplication
```

Comment:
For some reason, I had to manually add this line of code into the codegen.yml file. Is this supposed to be done manually?

Reply:
No, it's a manual thing unfortunately.

Reply:
No, it's a manual thing unfortunately.